.circleci/config.yml

```diff
@@ -64,16 +64,16 @@ jobs:
 workflows:
   build:
     jobs:
-      - lint
+      - lint:
+          context: Salsify
```

Comment:
We shouldn't add this context to CI for public projects since it won't work in forks.

Reply:
Oops yes, forgot to remove this when I swapped to use the `cimg` images.